src/modules/Dashboard/features/Applet/Popups/DuplicatePopups/DuplicatePopups.tsx

```diff
@@ -172,6 +172,31 @@ export const DuplicatePopups = ({ onCloseCallback }: { onCloseCallback?: () => v
     })();
   }, [duplicatePopupsVisible]);
 
+  useEffect(() => {
```

Comment:
I guess no need the local state `const [successModalVisible, setSuccessModalVisible] = useState(false);` anymore.

Could you please move this logic into a separate callback and use it instead of the trigger `setSuccessModalVisible(true);` in row 93;